api/db/migrate/20201227185349_create_plan_comments.rb

```diff
@@ -0,0 +1,10 @@
+ class CreatePlanComments < ActiveRecord::Migration[6.0]
+  def change
+    create_table :plan_comments do |t|
+      t.string :author, null: false
```

Comment:
We may in the future want to change this to the author's user id, but I think this is fine for now. Once we allow students to also make comments is when I think we'll need to change this.